packages/backend-modules/payments/lib/config.ts

```diff
@@ -13,12 +13,15 @@ export type Config = {
   REPUBLIK_STRIPE_ENDPOINT_SECRET: string
   MONTHLY_SUBSCRIPTION_STRIPE_PRODUCT_ID: string
   YEARLY_SUBSCRIPTION_STRIPE_PRODUCT_ID: string
+  BENEFACTOR_SUBSCRIPTION_STRIPE_PRODUCT_ID: string
   REPUBLIK_STRIPE_SUBSCRIPTION_TAX_ID: string
   REPUBLIK_STRIPE_PAYMENTS_CONFIG_ID: string
   PROJECT_R_STRIPE_PAYMENTS_CONFIG_ID: string
   REPUBLIK_3_MONTH_GIFT_COUPON: string
   PROJECT_R_3_MONTH_GIFT_COUPON: string
   PROJECT_R_YEARLY_GIFT_COUPON: string
+  PROJECT_R_REDUCED_MEMBERSHIP_DISCOUNTS: string[]
```

Comment:
can you add the new env variables to example.env? :)

Reply:
will do